coriolis/utils.py

```diff
@@ -498,3 +531,78 @@ def filter_chunking_info_for_task(task_info):
             vol["replica_state"]["chunks"] = "<redacted>"
 
     return cpy
+
+
+def _write_systemd(ssh, cmdline, svcname, run_as=None, start=True):
+    serviceFilePath = "/lib/systemd/system/%s.service" % svcname
+
+    if test_ssh_path(ssh, serviceFilePath):
+        return
```

Comment:
IMO we should try to backup the existing file and create a new one in case there are any differences.

Reply:
if we do that, we replace the file every time we call this function, whether it's needed or not, and reload systemd && restart application (which you will probably not want to do). If we need to replace a service, I propose we do it explicitly by removing the old config file and writing it again.